cmd/worker/main_test.go

```
@@ -367,6 +369,91 @@ func TestBridgeStreamResultsForwardsHeartbeat(t *testing.T) {
 	}
 }
 
+func TestBridgeStreamResultsForwardsMetadata(t *testing.T) {
+	t.Parallel()
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	max := 3
```

golangci-lint (2.11.4)

[error] variable max has same name as predeclared identifier

(predeclared)